.github/workflows/docker.yml

```diff
@@ -7,10 +8,10 @@ on:
       - 'main'
     tags:
       - 'v*.*.*'
-
 jobs:
   build:
     runs-on: ubuntu-latest
+    if: ${{ vars.QUAY_USERNAME != '' && vars.QUAY_ORG != '' }}
```

Comment:
Note that Github doesn't support conditions using secret values at all.  At the `jobs.<job_id>.if` level, it is legal to refer to `variables.`.  That was my motivation for moving the   QUAY_USERNAME from secret to variable.

https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability